forwarder/Cargo.toml

```diff
@@ -18,5 +18,8 @@ pedantic = "deny"
 as_conversions = "deny"
 
 [dependencies]
-forwarder-utils = {path = "../utils" }
-near-sdk.workspace = true
+arrayvec = { version = "0.7", default-features = false }
+borsh = { version = "1", default-features = false, features = ["derive"] }
+bs58 = { version = "0.5", default-features = false }
+hex = { version = "0.4", default-features = false }
+wee_alloc = { version = "0.4", default-features = false }
```

Comment:
Due to being unmaintained, it is not recommended to use `wee_alloc` any more. But do we need an allocator still? Looks like you have made a concerted effort to eliminate dynamic memory allocation entirely.

Reply:
Yes. I tried to get rid of the dependency. Because there is no dynamic memory allocation in the contract (at least it looks like this). I even tried to implement `core::alloc::GlobalAlloc` the `NoopAlloc`, but the contract doesn't work with such allocator.

Reply:
It looks like an allocation happens. But where, it's a question:
```
"Smart contract panicked: library/alloc/src/alloc.rs, at line: 418, at column: 13"
```

Reply:
I found this [allocation](https://github.com/aurora-is-near/aurora-forwarder-contractrs/blob/no_near_sdk/forwarder/src/runtime/io.rs#L90) 😃